recipes/qcodes/meta.yaml

```diff
@@ -0,0 +1,64 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "qcodes" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "a70f3a209e2f4371634b12094def37696fa2accb2f0746e5dffba50e4918e33e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://github.com/QCoDeS/Qcodes/archive/v{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - numpy >=1.10
+    - pyvisa >=1.8
+    - ipython >=4.1.0
+    - jupyter >=1.0.0
+    - ipywidgets >=4.1
+    - h5py >=2.6
+    - pathlib
+    - matplotlib
+    - coverage
+    - pyqtgraph
+
+test:
+  source_files:
```

Comment:
I believe `source_files` was introduced in conda-build 2.0.0, but currently conda-forge is using 1.21.14, which is causing an error. I'm not sure if there was some other mechanism for doing this in previous versions of conda-build. For now, you may just want to skip this step.